fuel-tests/tests/contract.rs

```diff
@@ -1,6 +1,7 @@
 use crate::helpers::{TestContext, TestSetupBuilder};
+use fuel_gql_client::client::{PageDirection, PaginationRequest};
 use fuel_vm::prelude::*;
-
+use rstest::rstest;
```

Comment:
keep imports separated from other vars

```suggestion
use rstest::rstest;

```